modules/openapi-generator/src/main/resources/Java/README.mustache

```diff
@@ -89,11 +89,14 @@ public class {{{classname}}}Example {
     public static void main(String[] args) {
         ApiClient defaultClient = Configuration.getDefaultApiClient();
         defaultClient.setBasePath("{{{basePath}}}");
-        {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
+        {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}{{#isBasicBasic}}
         // Configure HTTP basic authorization: {{{name}}}
-        HttpBasicAuth {{{name}}} = (HttpBasicAuth) defaultClient.getAuthentication("{{{name}}}");
```

Comment:
@addynaik may I know why "{{{name}}}" is removed from `getAuthentication()`?

Reply:
it should not be.. i think that was removed by accident. Thanks for catching that.

Reply:
@wing328 this should be fixed now